src/EnergyPlus/SizingManager.cc

```diff
@@ -3768,6 +3768,9 @@ namespace SizingManager {
 					thisTUFZSizing.DesHeatOAFlowFrac = 0.0;
 				}
 				thisTUFZSizing.MinOA = thisFZSizing.MinOA * minOAFrac;
+
+				// Make sure cooling min flow is at least the minOA flow, regardless of the term unit cooling sizing ratio applied above
+				thisTUFZSizing.DesCoolVolFlowMin = max( thisTUFZSizing.DesCoolVolFlowMin, thisTUFZSizing.MinOA );
```

Comment:
This seems like it stomps on things right at the end here.  Is there any chance this could result in a DesCoolVolFlowMin too high and have the same problem (ratio>1.0) downstream somewhere?

Reply:
Good question.  I added this, because the DOAS system in 5ZoneAirCooledWithDOASAirLoop came back with a zero design min frac because it's airterminal sizing spec is 0 for cooling 1.0 for outdoor air.  

One perspective is that the OA requirement should always be king at this point, so there should be two more lines here to make sure that DesHeatVolFlow and DesCoolVolFlow should be >=MinOA.  But then what about all the other flavors of that?  Will review the base zone sizing calcs to see which of these are checked against OA requirement in FinalZoneSizing.

Reply:
The other perspective is that the equipment should sort this out and this line should go.  The autosized airflow for this terminal unit was OK even without this line.  It was just the autosized min frac that showed as zero.

Reply:
@Myoldmopar @rraustad Back to sizing basics.  The zone MinOA requirement impacts the cooling design flow rate during the base zone sizing calcs [here](https://github.com/NREL/EnergyPlus/blame/fc8cc8d371830483fd2306f8a43b3fe2dea1d615/src/EnergyPlus/ZoneEquipmentManager.cc#L2717-L2745).  You can see from the blame that this has been the case since way-back-when.  Interesting that this is only applied to cooling flow, and not heating, but that's how it is.

The way the air terminal sizing spec is working at the moment, the OA requirement will be the floor for the design cooling flow rate, even if a particular air terminal is supposed to deliver a fraction (or zero) of the OA.  That's true with or without the line  that was added here to adjust DesignCoolVolMinFlow.

I'm working on a fix for that now.  I'm going to save some info about the impact of MinOA (like maybe DesignCoolVolFlowNoOA) that honors all the minima *except* MinOA and then use it here with the ratios to set TermUnitFinalZoneSizing.DesignCoolVolFlow appropriately.  Not sure about all the other flavors of \*Cool\* here.